### DISCUSSION TITLE
Contributing to Stacked

### DISCUSSION THREAD

**Discussion by CordMemescape** [FilledStacks/stacked#927] [Q&A] [answered]
I have several relatively minor contributions that I would like to make. Previously, before the monorepo was split out, creating a PR was relatively easy. However, with the recent changes, I'm not so clear on where or what to edit and how to submit. None of the "Contributing" guidelines contain any useful information on this topic yet.

The immediate area for contribution is the CLI (starting with the scale_on_hover.dart.stk template) followed by some fixes for the main Stacked package. Can anyone offer any guidance on how to start? For example,

1) Should we be editing the package/repo directly or should we use the development repo?
2) What are the preferred prefixes to use when creating a branch (fix:, feat:, etc.)?
3) What is the basic workflow we should follow (create branch/PR, update which files/changelogs, etc.)?
4) When editing CLI templates, what the workflow to edit and rebuild the compiled templates?
5) What else might be missing from this list?

Thanks!
Cord

**Comment by FilledStacks** [comment, marked as answer]
Hey @CordMemescape Adding this to my task tomorrow to write out something more comprehensive for contributors to follow along with. 

It's end of my day, but I'll have something tomorrow. I'll block an hour in my calendar for it now.

**Comment by CordMemescape** [comment]
Thanks Dane, much appreciated!
